app/utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import httpx
+
+async def fetch_temperature(city_name: str) -> float:
+    api_key = os.getenv("OPENWEATHER_API_KEY")

```

Comment:
Ensure that the environment variable `OPENWEATHER_API_KEY` is set correctly in your deployment environment. Without a valid API key, the request to the OpenWeather API will fail.